homeassistant/components/forecast_solar/strings.json

```diff
@@ -28,7 +28,8 @@
           "damping_evening": "Damping factor: adjusts the results in the evening",
           "inverter_size": "Inverter size (Watt)",
           "declination": "[%key:component::forecast_solar::config::step::user::data::declination%]",
-          "modules_power": "[%key:component::forecast_solar::config::step::user::data::modules_power%]"
+          "modules_power": "[%key:component::forecast_solar::config::step::user::data::modules_power%]",
+          "horizon": "Horizon profile as comma-separated list of degrees (0,0,25,30,20,10,0). See PVGIS."
```

Comment:
Same comment as was already linked, we should not ask for comma-separated input. That is not a good user experience.

Reply:
Ok, thank you. As Is aid, it's my first PR. I thought it was a good idea to implement it in the samw ay it's done in the API, the PVGIS tool and all other tools I checked when I tried to wrap my head around the horizon feature.
I'm sorry for causing you inconveniecne. I guess I'll eventually figure out how to get this functionality into homeassistant in some other way.